### DISCUSSION TITLE
issue state modification

### DISCUSSION THREAD

**Discussion by bougui** [mightymatth/arcli#16] [Ideas]
Hello,

your arcli tool is great to do some issue search or time log, will you try to add more option so that we can close some issue based on some filters ?

Best regards, Guillaume.

**Comment by mightymatth** [comment]
Thanks for the idea! do you have some examples of how would you like to use it? However, I need to investigate what API methods are available from Redmine so we can do that.

**Comment by bougui** [comment]
Hello,

let say that I would like to close all resolved issue that have not been updated since 14 days ... That would be great.

The resolved issue is an issue state in our redmine installations

And the update time is one of the filter available from the report pages.

Thanks.

**Comment by mightymatth** [comment]
When I get some spare time, I'll make a draft solution and you'll tell me what you think. 
If you have any other way to improve the CLI, you can start a discussion [here](https://github.com/mightymatth/arcli/discussions).

**Comment by bougui** [comment]
Great, let me know when your done ;-)
